requirements.txt

```diff
@@ -1 +1,2 @@
 PyZE==0.6.0
+renaut-api==0.0.1a0
```

Comment:
Why to you maintain a `requirement.txt` file ? It is not required in a custom component and I don't see any CI or tests that could make usage of this.

Reply:
Removed